src/main/resources/org/sagebionetworks/web/client/widget/discussion/DiscussionListWidgetViewImpl.ui.xml

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui" 
+	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+	<g:HTMLPanel>
```

Comment:
Instead of HTMLPanel, just use Div (or Row if it makes more sense for the layout)